e2e/pages/app/app.module.ts

```diff
@@ -1,10 +1,11 @@
 import { NgModule } from '@angular/core';
```

Comment:
When I run the tests i am getting 4 failures:

Sliders:
failed - should display tooltips
failed - should synchronize with text inputs

Item Display Panel:

failed - should display the panel upon clicking
failed - should be possible to use the close button to close the panel